app/src/main/res/layout/activity_main.xml

```diff
@@ -1,19 +1,244 @@
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_8"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_3"
+                    android:layout_column="@integer/column_1"
+            />
+            <Button
+                    android:text="@string/button_7"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_7"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_3"
+                    android:layout_column="@integer/column_0"
+            />
+
+            <Button
+                    android:text="@string/button_4"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_4"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_2"
+                    android:layout_column="@integer/column_0"
+            />
+            <Button
+                    android:text="@string/button_5"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_5"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_2"
+                    android:layout_column="@integer/column_1"
+            />
+            <Button
+                    android:text="@string/button_6"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_6"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_2"
+                    android:layout_column="@integer/column_2"
+            />
+
+            <Button
+                    android:text="@string/button_1"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_1"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_1"
+                    android:layout_column="@integer/column_0"
+            />
+            <Button
+                    android:text="@string/button_2"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_2"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_1"
+                    android:layout_column="@integer/column_1"
+            />
```

Comment:
put this un the last line, closing the button definition. In this case 
android:layout_column="@integer/column_1" />

Reply:
Done